src/auth/auth.controller.ts

```diff
@@ -38,4 +46,25 @@ export class AuthController {
   async register(@Body() userDto: CreateUserDto) {
     return this.authService.register(userDto);
   }
+
+  @Post("login")
+  @UseGuards(LocalAuthGuard)
+  @ApiOperation({
+    description:
+      "email or account as login account.  \n" +
+      "When the account information is correct will return token.",
```

Comment:
grammer recommmendation:

- Will return token when the account information is correct. (place result first will be better for reading)

Reply:
Revised.
[refactor: revised grammar questions about words](https://github.com/Cophr/backend/pull/10/commits/efd53265a8d42a602c74c96c85cd0166208335e9)